src/core/content/connector.ts

```diff
@@ -144,6 +144,14 @@ export default class BaseConnector {
 		alignItems: 'center',
 	};
 
+	/**
+	 * Function that gets a unique ID for channel/user blocklist.
+	 *
+	 * Has to be specified if usesBlocklist is set to true in connectors.ts.
+	 * If connectors.ts does not have usesBlocklist set to true, this should be null.
+	 */
+	public getChannelID: (() => string | null | undefined) | null = null;
```

Comment:
Should we call it `getChannelId` to be consistent? I know we have `getUniqueID` but this feels like a violation :/

Reply:
Done